var/spack/repos/builtin/packages/libxc/package.py

```diff
@@ -10,26 +10,20 @@ class Libxc(AutotoolsPackage):
     """Libxc is a library of exchange-correlation functionals for
     density-functional theory."""
 
-    homepage = "http://www.tddft.org/programs/octopus/wiki/index.php/Libxc"
-    url      = "http://www.tddft.org/programs/octopus/down.php?file=libxc/libxc-2.2.2.tar.gz"
-
-    version('4.3.2', sha256='bc159aea2537521998c7fb1199789e1be71e04c4b7758d58282622e347603a6f')
-    version('4.2.3', sha256='02e49e9ba7d21d18df17e9e57eae861e6ce05e65e966e1e832475aa09e344256')
-    version('3.0.0', sha256='5542b99042c09b2925f2e3700d769cda4fb411b476d446c833ea28c6bfa8792a')
-    version('2.2.2', sha256='6ca1d0bb5fdc341d59960707bc67f23ad54de8a6018e19e02eee2b16ea7cc642')
-    version('2.2.1', sha256='ade61c1fa4ed238edd56408fd8ee6c2e305a3d5753e160017e2a71817c98fd00')
-
-    patch('configure_add_fj.patch')
```

Comment:
Sorry, the description was wrong (I've corrected it). The patch has already been removed in #12860.

Reply:
Thanks for comment @adamjstewart . You can delete the patch file, because the purpose of implementing the patch was fulfilled by a modification of #12004 .

Reply:
> Sorry, the description was wrong (I've corrected it). The patch has already been removed in #12860.

It doesn't seem to me that #12860 has ever been merged. Can you just `git rm` the patch file if we don't need the patch anymore?

Reply:
I've removed the patch.